python/ray/data/_internal/execution/operators/actor_pool_map_operator.py

```diff
@@ -311,10 +311,13 @@ def base_resource_usage(self) -> ExecutionResources:
     def current_resource_usage(self) -> ExecutionResources:
         # Both pending and running actors count towards our current resource usage.
         num_active_workers = self._actor_pool.num_total_actors()
+        object_store_memory = self.metrics.obj_store_mem_cur
+        if self.metrics.obj_store_mem_pending is not None:
+            object_store_memory += self.metrics.obj_store_mem_pending
```

Comment:
(not blocking this PR. just a note.) 
I plan to move this method to `OpRuntimeMetrics`. Currently it's weird that we have 2 different places reporting resource metrics.